orangecontrib/text/widgets/owngramrange.py

```diff
@@ -0,0 +1,61 @@
+from Orange.widgets import settings
+from Orange.widgets import gui
+from Orange.widgets.widget import OWWidget
+from orangecontrib.text.corpus import Corpus
+from orangecontrib.text.widgets.utils import widgets
+
+
+class OWNgramRange(OWWidget):
+    name = "Ngram range"
+    description = "Transforms documents of a corpus to sets of token's ngrams."
+    icon = "icons/Ngrams.svg"
+    priority = 35
+
+    # Input/output
+    inputs = [("Corpus", Corpus, "set_data")]
+    outputs = [("Corpus", Corpus)]
+
+    want_main_area = False
+    resizing_enabled = False
+
+    ngram_range = settings.Setting((1, 1))
+    autocommit = settings.Setting(True)
+
+    def __init__(self):
+        super().__init__()
+        self.corpus = None
+
+        self.controlArea.layout().setMargin(5)
+
+        range_box = widgets.RangeWidget(self, 'ngram_range', minimum=1, maximum=10, step=1,
+                                        min_label='N-grams range:', dtype=int)
+        range_box.valueChanged.connect(self.on_change)
```

Comment:
To me more consistent with Orange's gui.py should we rather move this to a `callback` argument of `RangeWidget`?